native/acter/src/api/room.rs

```diff
@@ -226,8 +226,9 @@ impl Member {
                         <StoryEventContent as StaticEventContent>::TYPE,
                     ))
                 } else {
-                    // Not an acter space or story Posts are not activated..
-                    return false;
+                    // Story is less restricted than news
+                    // Everyone can post story as default
+                    return true;
```

Comment:
wait. no. this still needs to be activated and if there are no permissions set, it isn't to be understood as activated. this is false by  default, too.
```suggestion
                    return false;
```

Reply:
Oh then I need to add code in Space Setting >> Space Feature >> Stories, correct?

<img width="400" alt="Screenshot 2025-02-26 at 1 08 32 PM" src="https://github.com/user-attachments/assets/8b025649-a820-4815-8da9-0a9a8b5eca5c" />

<img width="389" alt="Screenshot 2025-02-26 at 1 08 54 PM" src="https://github.com/user-attachments/assets/92e4ba01-8145-4e4b-a4fd-e28943922690" />

Also need to add action button for Stories in Space Overview page?

<img width="418" alt="Screenshot 2025-02-26 at 1 09 24 PM" src="https://github.com/user-attachments/assets/2a13d735-1b44-4da9-b3f2-6927b9dde673" />

Reply:
@gnunicorn 
Can't we make stories as auto-activated as many people still doesn't know about this things?

Reply:
> Oh then I need to add code in Space Setting >> Space Feature >> Stories, correct?

Yes, we need to add the feature in the space settings as well. Same for allowing to create them from a space, yes.

> Can't we make stories as auto-activated as many people still doesn't know about this things?


No. We don't overrule the features people have activated when rolling out new features. The argument "but people don't know about it" would always apply to all new features but as we don't give admins any way to _disable_ this behavior beforehand, what gives us the right to impose the feature they should have activated in their existing spaces.

Now for spaces created _after_ we introduced stories, this will be different: once rolled out and considered working fine, we will make this (as well as other features) default on upon creation as part of the creation process, but it is something the creators _actively_ agree to. See https://balsamiq.cloud/sj2u1jx/plw5law .

Reply:
Got it, let me add required code in Space Settings as well as Space Overview screen.

Reply:
Done with below commits.

### 1) Add Story action button in Space Overview screen:
https://github.com/acterglobal/a3/pull/2537/commits/f9a4bd4fc19c41de925a87913afb31c5442e0541
![Screenshot 2025-02-27 at 3 00 48 PM](https://github.com/user-attachments/assets/32203f39-5afc-4b40-a078-17e3cc76c678)


### 2) Added stories activation option in Space Feature settings:
https://github.com/acterglobal/a3/pull/2537/commits/75b43d9f69a238c60914a910015058943d30177a
![Screenshot 2025-02-27 at 3 01 43 PM](https://github.com/user-attachments/assets/56655f04-9605-4a57-ad96-b35e45c7d73f)
![Screenshot 2025-02-27 at 3 01 55 PM](https://github.com/user-attachments/assets/87bc002f-509e-4ba6-b063-4dffd1e48964)